### DISCUSSION TITLE
SSO not working for On premise app with Azure AD

### DISCUSSION THREAD

**Discussion by karthikcefsharp** [cefsharp/CefSharp#4552] [General]
We are trying to access on-premise application using Azure-AD as identity provider to achieve single sign-on to Azure App proxy. When we are passing the token and required cookies in Cefsharp instance, we are unable to load/render the application properly. Though postman query we are able to achieve this. We need to understand the details why it's breaking in the Cefsharp. If you can provide a sample example on this would be great.

**Comment by amaitland** [comment]
Firstly this isn't a `CefSharp` specific issue. Make sure you read the first few paragraphs of https://github.com/cefsharp/CefSharp/wiki/General-Usage to get a better understanding of the project.

Historically Azure AD auth was implemented as an extension (https://www.magpcss.org/ceforum/viewtopic.php?f=6&t=18560&p=49675). 

There's possibly a new option as suggested by https://www.magpcss.org/ceforum/viewtopic.php?f=6&t=19517&p=53899 I'd suggest joining the discussion there on ceforum. I've not had a need to try this myself.

See https://github.com/cefsharp/CefSharp/discussions/4123 for some generic background on CEF's `Chrome Runtime`.
